command/state_mv.go

```diff
@@ -353,13 +353,19 @@ func (c *StateMvCommand) Run(args []string) int {
 		}
 	}
 
-	c.showDiagnostics(diags)
-
 	if moved == 0 {
-		c.Ui.Output("No matching objects found.")
-	} else {
-		c.Ui.Output(fmt.Sprintf("Successfully moved %d object(s).", moved))
+		diags = diags.Append(tfdiags.Sourceless(
+			tfdiags.Error,
+			"Invalid target address",
```

Comment:
I should've noted this in prior review, but `msgInvalidTarget` is a constant in this file we can use instead of this string literal